package.json

```diff
@@ -1,9 +1,11 @@
 {
   "name": "ideal-synergy",
   "version": "1.0.0",
+  "type": "module",
   "description": "",
   "main": "index.js",
   "scripts": {
+    "start":"node –experimental-modules ./index.js",
```

Comment:
This will be not user - please remove this line

Reply:
Fixed